webpack/webpack.renderer.config.js

```diff
@@ -132,25 +132,8 @@ module.exports = ({ production })  => {
                   'sass-loader']
             },
             {
-              test: /\.(png|jpe?g|gif)(\?.*)?$/,
-              use:
-                {
-                  loader: 'url-loader',
-                  options: {limit: 10240, name: 'imgs/[name]--[folder].[ext]'}
-                }
-            },
-            {
-              test: /\.(mp4|webm|ogg|mp3|wav|flac|aac)(\?.*)?$/,
-              loader: 'url-loader',
-              options: {limit: 10240, name: 'media/[name]--[folder].[ext]'}
-            },
-            {
-              test: /\.(woff2?|eot|ttf|otf)(\?.*)?$/,
-              use:
-                {
-                  loader: 'url-loader',
-                  options: {limit: 10240, name: 'fonts/[name]--[folder].[ext]'}
-                }
+              test: /\.ttf/,
```

Comment:
I don't understand this; we remove the special cases for a many extensions, but only have a special case left for ttf?  Is that the only extension that needs handling now?  wav|flac|png|woff2 all work fine without checks, but ttf still needs help?

Reply:
`wav|flac|png|woff2` _would not_ work without checks, but we don't use them. The only one of those extensions we're actually using in the repo is `ttf`.

The idea is to keep the config file as simple as possible. And if you consider the config file as a type of documentation in and of itself, these `module.rules` entries denote what to expect to find in the repository.

Reply:
So it relates to 'asset' (fonts / icons) being loaded, and the only assets that we use are fonts?

Reply:
how were they actually loaded?  was a URL to some file in the repo constructed?  What happens differently now?

Reply:
TTF = "TrueType font" = font files. We have one non-standard font we load into the app, _Nunito_, which was not being loaded correctly.

I'm not entirely sure how asset loading worked / works... The big difference now is that webpack does all of the work for us and we no longer need the special loader plugins.